app/models/search_facet.rb

```diff
@@ -164,10 +166,10 @@ class SearchFacet
     end
     property :filters do
       key :type, :array
-      key :description, 'Array of filter values for facet'
+      key :description, 'Array of filter values for facet (will default to public_filters if no user is signed in)'
```

Comment:
Blocker: there seems to be a subtle documentation bug that substantially changes meaning.

```suggestion
      key :description, 'Array of filter values for facet (will default to public_filters if user is not signed in)'
```